testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -89,6 +89,16 @@ def test_Universe_topology_IE_msg():
             raise AssertionError
 
     @staticmethod
+    def test_Universe_filename_IE_msg():
```

Comment:
Could you test reading a file that exists but is broken in some way so it will raise an `IOError`?